common/client-libs/validator-client/src/lib.rs

```diff
@@ -185,6 +210,11 @@ impl Client {
         Ok(mixnodes)
     }
 
+    pub async fn get_cached_mix_nodes(&mut self) -> Result<Vec<MixNodeBond>, ValidatorClientError> {
```

Comment:
Any reason this methods (as well as all other validator methods) is taking `&mut self`? `query_validators` does not require a mutable reference

Reply:
Good catch. No really sure how I ended with a `mut` here, but it is definitely not needed. Thanks.